# 33. SDG Parameters & Future Projections (EU 2030 / PNRR)
This notebook maps Italy's current 2025/2026 reality against the **UN Sustainable Development Goal 4 (Quality Education)** parameters and the **EU 2030 / PNRR (National Recovery and Resilience Plan) targets**.

By establishing a mathematical benchmark of "Prospected Future vs Current Reality", we create a baseline for future data analysis to track whether Italy actually meets its binding international obligations.

In [1]:
import pandas as pd
import os

os.makedirs('local_data/processed/macro_constraints', exist_ok=True)

### 1. UN SDG 4 & EU 2030 Benchmarks vs Current Reality
We compare the binding 2030 targets set by the European Union and the UN against Italy's current trajectory.

In [2]:
sdg_data = {
    'Metric': ['NEET Rate (15-29)', 'Early School Leavers (18-24)', 'Tertiary Education Attainment (25-34)'],
    'EU_2030_Target_Pct': ['< 9.0%', '< 9.0%', '> 45.0%'], # Binding EU targets
    'Italy_Current_Reality_2025_Pct': ['13.3%', '10.5%', '29.2%'], # Current Eurostat/ISTAT baselines
    'Trajectory_Status': ['Severely Off-Track', 'Off-Track', 'Critically Off-Track']
}
df_sdg = pd.DataFrame(sdg_data)
df_sdg.to_csv('local_data/processed/macro_constraints/sdg_2030_targets_vs_reality.csv', index=False)
df_sdg

,Metric,EU_2030_Target_Pct,Italy_Current_Reality_2025_Pct,Trajectory_Status
0,NEET Rate (15-29),< 9.0%,13.3%,Severely Off-Track
1,Early School Leavers (18-24),< 9.0%,10.5%,Off-Track
2,Tertiary Education Attainment (25-34),> 45.0%,29.2%,Critically Off-Track


### 2. PNRR Future Projections (National Recovery and Resilience Plan)
Italy was granted massive EU funds (PNRR) to fix infrastructure. We model the prospected targets vs the execution reality.

In [3]:
pnrr_data = {
    'Intervention_Area': ['New School Buildings', 'Childcare/Nurseries (Asili Nido)', 'School Canteens (Tempo Pieno)'],
    'PNRR_Allocated_Funds_Billion_EUR': [1.19, 4.60, 0.96],
    'Target_Goal': ['Build 212 highly sustainable schools', 'Create 264k new nursery slots', 'Build 1000 new canteens for full-time school'],
    'Current_Execution_Risk': ['High (Inflation/Delays)', 'Critical (Municipalities unable to co-fund/staff)', 'High (South lacks infrastructure)']
}
df_pnrr = pd.DataFrame(pnrr_data)
df_pnrr.to_csv('local_data/processed/macro_constraints/pnrr_projections_vs_risk.csv', index=False)
df_pnrr

,Intervention_Area,PNRR_Allocated_Funds_Billion_EUR,Target_Goal,Current_Execution_Risk
0,New School Buildings,1.19,Build 212 highly sustainable schools,High (Inflation/Delays)
1,Childcare/Nurseries (Asili Nido),4.60,Create 264k new nursery slots,Critical (Municipalities unable to co-fund/staff)
2,School Canteens (Tempo Pieno),0.96,Build 1000 new canteens for full-time school,High (South lacks infrastructure)


### 3. Conclusion
Italy's current reality is statistically decoupled from the SDG 4 and EU 2030 targets. With a tertiary attainment rate of barely 29.2% (vs the 45% target) and a NEET rate of 13.3% (vs the 9% target), the structural constraints (Debt, Demographics) mathematically prevent the system from reaching the "Prospected Future" without radical intervention.